deploy/sre-prometheus/100-managed-upgrade-operator.PrometheusRule.yaml

```
@@ -0,0 +1,77 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: sre-managed-upgrade-operator-alerts
+    role: alert-rules
+  name: sre-managed-upgrade-operator-alerts
+  namespace: openshift-monitoring
+spec:
+  groups:
+  - name: sre-managed-upgrade-operator-alerts
+    rules:
+    - alert: UpgradeConfigValidationFailed
```

We seem to have an existing naming convention of appending "SRE" to our alerts, I guess to distinguish them from built-in k8s/OCP alerts. Should we do similar for these?

On a related note, what do you think about establishing a common naming convention so that they're easier to allow-filter in the silences we create? ie. making our silence allow anything matching a regex of "Upgrade.*SRE". cc @sedroche 

```suggestion
    - alert: UpgradeConfigValidationFailedSRE
```